game/view/sector.lua

```diff
@@ -211,6 +214,37 @@ function SectorView:getBodySprite(body)
   return body_sprite
 end
 
+
+function SectorView:setBodySprite(body, draw)
+  self.body_sprites[body:getId()] = draw
+end
+
+function SectorView:getBodyDialogue(body, i, j, pi, pj)
+  local id = body:getId()
+
+  --Get appropriate position for dialogue box
+  local side
+  if pj <= j then
+    side = "right"
+  else
+    side = "left"
+  end
```

Comment:
WTF is pi and pj? Please, prefer being verbose than unclear

Reply:
I will change it to player_i and player_j